File: Latihan_b/cpp/Hardware.cpp
#include "Product.cpp"

class Hardware : public Product{
    private:
        string brand;
        string model;

    public:
        Hardware(string brand, string model)
        {
            this->brand = brand;
            this->model = model;
        }

        Hardware(){}
        
        void setBrand(string brand)
        {
            this->brand = brand;
        }
        string getBrand()
        {
            return this->brand;
        }

        void setModel(string model)
        {
            this->model = model;
        }

        string getModel()
        {
            return this->model;
        }

        ~Hardware(){}
};

File: Latihan_b/py/Hardware.py
from numpy import product
from Product import Product

class Hardware(Product):
    __brand = ""
    __model = ""

    def __init__(self):
        self.__brand = ""
        self.__model = ""

    def setBrand(self, brand):
        self.__brand = brand

    def getBrand(self):
        return self.__brand

    def setModel(self, model):
        self.__model = model

    def getModel(self):
        return self.__model